Sources/SafeDICore/Generators/DependencyTreeGenerator.swift

```diff
@@ -39,7 +39,7 @@ public final class DependencyTreeGenerator {
 
         let typeDescriptionToScopeMap = try createTypeDescriptionToScopeMapping()
         try validateReceivedProperties(typeDescriptionToScopeMap: typeDescriptionToScopeMap)
-        let rootScopeGenerators = try rootInstantiableTypes
+        let rootScopeGenerators = try rootInstantiables
```

Comment:
find/replace casualty but I don't mind it.

Reply:
I don't either. I also like now that you can return `Instantiable` instead of `InstaniableType` elsewhere.